### DISCUSSION TITLE
Profile photo

### DISCUSSION THREAD

**Discussion by morzamohammaf** [community/community#42592] [Mobile]
Hi sir how to add profile photo in GitHub Mobile?

**Comment by DSDmark** [comment]
Hi @morzamohammaf,

I guess, There is no that type of feature in GitHub mobile yet. But, you could use mobile browser for that purpose.

**Reply by morzamohammaf** [reply]
> Hi @morzamohammaf,
> 
> I guess, There is no that type of feature in GitHub mobile yet. But, you could use mobile browser for that purpose.

OK sir thank you, sorry for late reply

**Reply by morzamohammaf** [reply]
OK mam, thank you,sorry for late reply

**Reply by morzamohammaf** [reply]
Please mam guide me, is it possible to access and use "GitHub packages" in GitHub Mobile? If it is, then what is the method?

**Reply by morzamohammaf** [reply]
Sorry sir I am really sorry, I have made a great mistake, please sir forgive me.

**Reply by DSDmark** [reply]
@morzamohammaf, Can you describe more about what you mean by `GitHub packages`?. And also if you want to help. Then, don't hesitate to ask, we are here for helping each other.

**Reply by morzamohammaf** [reply]
GitHub packages: yes sir  before I installed GitHub Mobile app, actually I created a GitHub account using chrome, and there was also mentioned  "packages" as well as        repositories, pull request and issues in the homepage. and on click packages, I found many things like npm, node.js and vs.code etc. But in GitHub Mobile, I am unable to  find all these things. I mean how can I find all these things in GitHub Mobile app?

**Reply by radar-blip** [reply]
It could be that GitHub Mobile App does not have all the features that the GitHub Web UI has.
So, if you went back to using Chrome, those features would be available, again. 😃

**Reply by morzamohammaf** [reply]
> It could be that GitHub Mobile App does not have all the features that the GitHub Web UI has.
> So, if you went back to using Chrome, those features would be available, again
    Hello sir, I need help please help me like before. Sir I have been signe out from my account, and I am unable to sign in to the account. I have tried also in"GitHub support". But github support send me a "ticket Id"no: and nothing more. I also don't know what is the ticket Id no: used for.sir please help me in this matter.

**Comment by morzamohammaf** [comment]
Yes sir I want to use chrome to find all those features, but there also I
am facing problems, I don't know why my account homepage is not opening
completely. When I signe in to the account, always I find there,
repositories are loading, and that scene never finish.😂

**Reply by radar-blip** [reply]
Sounds like you are looking for something that's never there.
And waiting for something that never will load. 
I wonder what the problem is. 😕

**Reply by DSDmark** [reply]
If you were talking about chrome in mobile. Then, you could try to enable the **Desktop Site**. It's suck some time when you're using mobile.
![](https://browserstack.wpenginepowered.com/wp-content/uploads/2022/01/Screenshot1.png)

**Reply by morzamohammaf** [reply]
Hi sir thank you so much for guiding me in a very nice and clear way.after discuss my problem with you, now I am able to get back in my account again. I was going to reply you at the moment, but because of network issue I am replying late.God bless you sir.

**Comment by vivek09thakur** [comment]
Login in to GitHub in browser.  Tap on your profile you'll get many option then choose edit personal profile upload your profile picture.

**Reply by morzamohammaf** [reply]
Ok sir thank you so much.

**Reply by morzamohammaf** [reply]
> Login in to GitHub in browser.  Tap on your profile you'll get many option then choose edit personal profile upload your profile picture. 

Sir help me please, I have been signe out from my account, and I have tried many times in "GitHub support" to solve this problem, but github support send me a ticket Id no: and nothing more. I also don't know, what is the ticket Id no: used for. Sir please help me, if there is any solution for this problem.

**Reply by DSDmark** [reply]
@morzamohammaf, Here is the documentation how to use GitHub ticket ID:- https://docs.github.com/en/support/contacting-github-support/viewing-and-updating-support-tickets.

And about sign out things, can you add screenshot for better understanding. What the problem really is.

**Comment by morzamohammaf** [comment]
Ok, I'll check it out.

**Reply by morzamohammaf** [reply]
> Ok, I'll check it out.
> 
> On Sat., 31 Dec. 2022, 6:39 pm Sanskar Sahu, ***@***.***>
> wrote:
> 
> > Here is the documentation how to use GitHub ticket ID:-
> > https://docs.github.com/en/support/contacting-github-support/viewing-and-updating-support-tickets
> > .
> >
> > And about sign out things, can you add screenshot for better
> > understanding. What the problem really is.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/orgs/community/discussions/42592#discussioncomment-4560379>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/A4HVZLTBNQUG3Y4ISJAKY3TWQAZQHANCNFSM6AAAAAATI3RXFY>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
> 

No sir I can't add screenshot, but I can describe more, that as I try to sign in, there appears "404" "this is not the webpage you are looking for""
